client-side/src/index.js

```diff
@@ -3,11 +3,17 @@ import ReactDOM from 'react-dom/client';
 import './index.scss';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
change imports order, first external modules, and then internal files

Reply:
ok